pkg/occlient/occlient_test.go

```diff
@@ -6,10 +6,83 @@ import (
 	"os"
 	"reflect"
 	"testing"
+
+	routev1 "github.com/openshift/api/route/v1"
+	ktesting "k8s.io/client-go/testing"
 )
 
-func TestGetOcBinary(t *testing.T) {
+func TestCreateRoute(t *testing.T) {
+	// initialising the fakeclient
+	fkclient, fkclientset := FakeNew()
```

Comment:
this should be initialized for every test case separately

Reply:
You mean sub-tests? `TestCreateRoute` is a test case, no?

Reply:
yes sub-test.
As it is currently written every test case reuses the same client.
That means that state from one test case can effect other.